From 452e703fca2a4dc2fee1b5d5e6a6a9e1c6e7a933 Mon Sep 17 00:00:00 2001
From: Ramulu Komme <rkomme@maxlinear.com>
Date: Fri, 19 May 2023 15:36:58 +0530
Subject: [PATCH] framework: platform/bpl: Add missing bpl platform sources

prplmesh compilation breaks for OpenWRT UGW platform due to
missing bpl platform sources. Add missing bpl platform
sources so that prplmesh compiles for OpenWRT UGW platform.

https://prplfoundationcloud.atlassian.net/browse/PPM-2540

Signed-off-by: Ramulu Komme <rkomme@maxlinear.com>
---
 framework/platform/bpl/CMakeLists.txt | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/framework/platform/bpl/CMakeLists.txt b/framework/platform/bpl/CMakeLists.txt
index b3819628e..5eff4784f 100644
--- a/framework/platform/bpl/CMakeLists.txt
+++ b/framework/platform/bpl/CMakeLists.txt
@@ -64,7 +64,12 @@ if (TARGET_PLATFORM STREQUAL "openwrt")
 
         # UCI BPL
         file(GLOB_RECURSE bpl_platform_sources 
-            ${MODULE_PATH}/*/uci/*.c*
+            ${MODULE_PATH}/arp/linux/*.c*
+            ${MODULE_PATH}/board/uci/*.c*
+            ${MODULE_PATH}/cfg/uci/*.c*
+            ${MODULE_PATH}/common/uci/*.c*
+            ${MODULE_PATH}/db/uci/*.c*
+            ${MODULE_PATH}/dhcp/uci/*.c*
             ${MODULE_PATH}/cfg/vendor/*.c*
         )
 
-- 
2.17.1

